README.md

```diff
@@ -260,6 +261,43 @@ it.
 NOTE: You must ensure that the data type (e.g. Object, Array or String) for
 the replaced value is correct.  This cannot be done automatically.
 
+Hjson Files
+----------
+
+Hjson is a syntax extension to JSON. It is intended to be used like a user interface for humans, to read and edit before passing the JSON data to the machine. That means you can use it to parse JSON files but it is not intended as a replacement.
+
+You can check [Hjson's homepage](https://hjson.org) to get familiar with it and you can [try out it's syntax](https://hjson.org/try.html).
+
+Main features:
+- Comments
+- Optional quotes
+- Optional commas
+- Heredoc
+
+Example syntax
+
+```
```

Comment:
GFM supports syntax highlighting. It doesn't appear to support HJSON but JS looks to be an improvement: 

```js
{
 +    # specify rate in requests/second (because comments are helpful!)
 +    rate: 1000
 +
 +    // prefer c-style comments?
 +    /* feeling old fashioned? */
 +
}
```

Reply:
I tried that but if you use whole example it is messed up due to an apostrophe.
```js
{
    # specify rate in requests/second (because comments are helpful!)
    rate: 1000

    // prefer c-style comments?
    /* feeling old fashioned? */

    # did you notice that rate doesn't need quotes?
    hey: look ma, no quotes for strings either!

    # best of all
    notice: []
    anything: ?

    # yes, commas are optional!
}
```

Reply:
Oh, ok I see what you did there. I am going to change it.